### PR TITLE
fix: error TS6369: Option '--build' must be the first command line argument.

### PR DESCRIPTION
**Pull request by llllvvuu**
Problem
---
When we use `--signalEmittedFiles` it will insert `--listEmittedFiles` and then `tsc` will complain

> Option '--build' must be the first command line argument.

Solution
---
Extract `-b`/`--build` and move it before `--listEmittedFiles`.

### CODE REVIEW

**Comment by gilamran** [comment]
Looking good, but can you please add a test on `args-manager.test.ts`?

**Comment by llllvvuu** [comment]
OK, I moved it to `args-manager` and added a test there

File: src/lib/tsc-watch.ts [hunk 1]
```diff
@@ -125,7 +125,6 @@ function spawnTsc({ maxNodeMem, requestedToListEmittedFiles, signalEmittedFiles
   const nodeArgs = [
     ...((maxNodeMem) ? [`--max_old_space_size=${maxNodeMem}`] : []),
     tscBin,
-    ...((signalEmittedFiles || requestedToListEmittedFiles) ? ['--listEmittedFiles'] : []),
```
Comment (gilamran) [inline src/lib/tsc-watch.ts:128]: why was this removed?

Comment (llllvvuu) [reply src/lib/tsc-watch.ts:128]: I moved this to `args-manager.ts` so the order of `--build` vs `--listEmittedFiles` can be tested

**Comment by gilamran** [comment]
Published on version 6.0.5
Thanks!
